### PR TITLE
Add minio attributions to newest versions of Konvoy, Kommander, and Kaptain

### PR DESCRIPTION
**Pull request by kevinmiller-conley**
## Jira Ticket

Urgent ask from legal and Tobi

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4453.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cneth — APPROVED** [review]

**Review by gvasquezvargas — COMMENTED** [review]
Made tiny changes to make Vale happy. 
Do we have the same issues with MinIO in 2.3? If yes, we need to add to those pages as well.

File: pages/dkp/kommander/2.2/download/index.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ To download a new version of DKP, you have 2 options:
 You must be a registered user and logged on to the support portal to download DKP. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install this product.
 If you have problems downloading DKP, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.</p>
 
+<p class="message--note"><strong>NOTE: </strong>This docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. For more information, refer to <a href="https://github.com/minio/minio">MinIO's source code</a>.</p>
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/download/index.md:23]: We can't do this. @kevinmiller-conley was this explicitly called for?

**Comment by cmastr** [comment]
Please pause until we can talk with outside council.

**Review by cmastr — CHANGES_REQUESTED** [review]
let's wait to talk to outside council.

File: pages/dkp/kommander/2.2/download/index.md (hunk 1 above)
Comment (msdolbey) [inline pages/dkp/kommander/2.2/download/index.md:23]: Docker is caps'ed

```suggestion
<p class="message--note"><strong>NOTE: </strong>This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. For more information, refer to <a href="https://github.com/minio/minio">MinIO's source code</a>.</p>
```

File: pages/dkp/kommander/2.2/install/air-gapped/index.md [hunk 2]
```diff
@@ -163,6 +163,8 @@ See [Kommander Load Balancing][kommander-load-balancing] for more information.
 
 ### Load the Docker images into your Docker registry
 
+<p class="message--note"><strong>NOTE: </strong>These docker images include code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. For more information, refer to <a href="https://github.com/minio/minio">MinIO's source code</a>.</p>
```
Comment (msdolbey) [inline pages/dkp/kommander/2.2/install/air-gapped/index.md:166]: Docker is caps'ed

```suggestion
<p class="message--note"><strong>NOTE: </strong>These Docker images include code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. For more information, refer to <a href="https://github.com/minio/minio">MinIO's source code</a>.</p>
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/seed-a-registry/index.md [hunk 3]
```diff
@@ -16,7 +16,8 @@ Before creating a Kubernetes cluster you must have the required images in a loca
     ```bash
     curl -o konvoy-image-bundle.tar.gz -O https://downloads.d2iq.com/dkp/v2.2.0/konvoy_image_bundle_v2.2.0_linux_amd64.tar.gz
     ```
-<p class="message--note"><strong>NOTE: </strong>This docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. Complete source code for MinIO is available <a href="https://github.com/minio/minio">here</a>.</p>
+
+    <p class="message--note"><strong>NOTE: </strong>This docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. For more information, refer to <a href="https://github.com/minio/minio">MinIO's source code</a>.</p>
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/seed-a-registry/index.md:20]: Docker is caps'ed

```suggestion
    <p class="message--note"><strong>NOTE: </strong>This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. For more information, refer to <a href="https://github.com/minio/minio">MinIO's source code</a>.</p>
```

File: pages/dkp/konvoy/2.2/download/index.md [hunk 4]
```diff
@@ -10,6 +10,8 @@ enterprise: false
 
 <!-- markdownlint-disable MD034 -->
 
+<p class="message--note"><strong>NOTE: </strong>This docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. For more information, refer to <a href="https://github.com/minio/minio">MinIO's source code</a>.</p>
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.2/download/index.md:13]: Docker is caps'ed

```suggestion
<p class="message--note"><strong>NOTE: </strong>This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. For more information, refer to <a href="https://github.com/minio/minio">MinIO's source code</a>.</p>
```

**Review by cmastr — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

**Pull request merged**